gensim/models/nmf.py

```diff
@@ -93,7 +93,8 @@
 
 """
 
-import collections
+
+import collections.abc as collections
```

Comment:
Is there any real benefit in doing this instead of:

```suggestion
import collections.abc
```

You're saving 4 characters, but making it less obvious what's actually being used.

Reply:
I was thinking of keeping small diffs at the time but I agree that it would make more sense to change it

Reply:
Yes, that is a somewhat valid reason, but think about the trade-off:

1. We do a bit more work during this review (one-off cost)
2. People reading the code may be confused (repetitive cost)

Better to pay once up front and save future generations of gensim developers the pain (I'm exaggerating, obviously, but you probably get my point).

Reply:
Yeah absolutely! I'll get that changed. Hopefully I'll have it sorted in a couple hours